specs/state-proof.md

```diff
@@ -79,10 +83,12 @@ to not be in the table.
 - 6.2. `value` is 0 if first access and READ
 - 6.3. `initial value` is 0
 - 6.4. `state root` is the same
+- 6.5. `value_prev` is 0
 
 ### Account
```

Comment:
Nitpicking: we could update the code so that the constraints "blocks" (account, tx refund...) are in the same order (and make sure all are numbered) as in the specs